src/Message/AuthorizeRequest.php

```diff
@@ -7,6 +7,21 @@
  */
 class AuthorizeRequest extends AbstractRequest
 {
+    public function getApplicationFee()
+    {
+        return $this->getParameter('applicationFee');
+    }
+
+    public function getApplicationFeeInteger()
+    {
+        return (int) round($this->getApplicationFee() * pow(10, $this->getCurrencyDecimalPlaces()));
```

Comment:
I would rather not have the two functions. Is there a reason for it? If rounding is absolutely necessary, why not just do it in the getApplicationFee() method?